src/node_api.cc

```diff
@@ -3088,6 +3088,18 @@ napi_status napi_get_version(napi_env env, uint32_t* result) {
   return napi_clear_last_error(env);
 }
 
+napi_status napi_get_node_version(napi_env env,
+                                  uint32_t* versions,
```

Comment:
Would it be better to use a struct here?  I see two benefits:
* No need for the developer to study the documentation to determine the size of the array required
* Easier to access the parts of the version by member name